realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -45,7 +46,7 @@
  * @see io.realm.Realm
  * @see io.realm.DynamicRealm
  */
-abstract class BaseRealm implements Closeable {
+public abstract class BaseRealm implements Closeable {
```

Comment:
Couldn't we move RealmObjectProxy to `io.realm` instead and make it package private? That would be less annoying than having to make BaseRealm public?